versioned_docs/version-8.2/components/modeler/desktop-modeler/element-templates/about-templates.md

```diff
@@ -8,7 +8,7 @@ description: "Element templates are a way to extend Camunda Modeler with domain-
 Element templates are currently available in BPMN diagrams only.
 :::
 
-Element templates are a way to extend the [modeler](https://camunda.org/bpmn/tool/) with domain-specific diagram elements, such as service and user tasks.
+Element templates are a way to extend the [modeler](https://camunda.com/download/) with domain-specific diagram elements, such as service and user tasks.
```

Comment:
Page would likely be the modeler downloads page, however we'd need to be aware of [this](https://camunda.slack.com/archives/C034F8NA1G8/p1682497416851919).

```suggestion
Element templates are a way to extend the [modeler](https://camunda.com/download/modeler/) with domain-specific diagram elements, such as service and user tasks.
```

Reply:
Okay, will adjust.